.travis.yml

```diff
@@ -1,16 +1,17 @@
 sudo: false
 language: node_js
 node_js:
-  - '0.10'
+  - '5.10.1'
```

Comment:
Let's discuss this node.js 5.x stuff some more. I have been advocating the node 4.x version with LTS (Long Term Support). It is a little more stable. I'm not against it but I haven't used it because its more of the bleeding edge. And its really only used for build tools. I'll push this through now though.